queries/kotlin/highlights.scm

```diff
@@ -1,36 +1,44 @@
-;;; Kotlin 
+;;; Kotlin
 ;; Issues
 
 ; mapOf with type definition AND multiple entries are not highligted correctly. Issue in parser. Eg.
 ; a = mapOf<String, String>(AA.value to AA.value, AA.value to AA.value)
 
+;; parser does not recognice `in` and `downto` in `for (i in 5..1) print(i)`
 
-(function_modifier) @type
-
-;; String
+;; Strings
 
 (line_string_literal) @string
 (multi_line_string_literal) @string
-(line_string_literal (interpolated_identifier)) @string
-(line_string_literal (interpolated_expression (line_string_literal))) @string
+; Interpolated
+(interpolated_identifier) @none
```

Comment:
can be `(interpolated_identifier) @none @variable` (default has not highlighting set for variable, but some users set colors for basic identifiers)